src/pfe/file-watcher/server/test/resources/workspace.json

```diff
@@ -0,0 +1,67 @@
+{
+	"apiVersion": "1.0.0",
+	"metadata": {
+		"name": "codewind-plugin-env"
+	},
+	"projects": [
```

Comment:
I will remove all the projects from dev file. We will clone projects like local because Portal does initializing, ie; they modify charts when we import.. so we need to clone for Kube, we cannot bind with template projects because of this